src/components/SocialButtons/SocialButton.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import Button from '../common/button/Button';
+// import Footer from '../footer/Footer';
+// import NavBar from '../navbar/navbar';
+
+
+const SocialButtons = () => {
+  const { BACKEND_URL } = process.env;
```

Comment:
Use the URL in the action types as we agreed

Reply:
resolved